typescript/db/src/repository/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Prisma } from "@prisma/client";
+import { Prisma } from "../__generated__";
```

Comment:
I get an error from eslint here: `../__generated__ import should occur after import of slugify `